wpilib/wpilib/speedcontrollergroup.py

```diff
@@ -26,9 +26,15 @@ def __init__(self, *args):
         SendableBase.__init__(self)
         SpeedController.__init__(self)
 
-        for speedController in args:
-            self.addChild(speedController)
-        self.speedControllers = args
+        self.speedControllers = []
+        self.speedControllers.append(speedController)
+        self.addChild(speedController)
+
+        for speedcontroller in args:
+            self.speedControllers.append(speedcontroller)
```

Comment:
tbh I would've preferred `self.speedControllers = [speedController] + args` but this works as well I guess

Reply:
This is closer to what they have.

Reply:
True, but only because that's how you have to do it in Java. You might notice the C++ implementation is much more succinct.

Reply:
Should probably also change `args` to `speedControllers` too. Hmm.